module/PowerNSX.psm1

```diff
@@ -27667,6 +27667,33 @@ function Set-NsxLoadBalancerPoolMember {
 
     This cmdlet configures the state of the specified LoadBalancer Pool Member.
 
+    .EXAMPLE
+    Get-NsxEdge testedge | Get-NsxLoadBalancer | Get-NsxLoadBalancerPool pool1 |
```

Comment:
Im not really concerned about the name of the edge, as long as the example is valid.  Not sure what you are asking in the second comment.